frontend/app/controllers/Joiner.scala

```diff
@@ -80,7 +80,7 @@ object Joiner extends Controller with ActivityTracking
       contentURL <- contentRefererOpt if Uri.parse(contentURL).host.contains("www.theguardian.com")
       access <- accessOpt
     } yield {
-      Redirect(routes.MemberOnlyContent.membershipContent(contentURL, access.name))
+      Redirect(routes.MemberOnlyContent.membershipContent(Some(contentURL)))
```

Comment:
Probably legacy code at this point - is anyone still directing users to /tier-chooser as a content-access handler?